internal/controller/instaslice_daemonset_test.go

```diff
@@ -86,8 +86,12 @@ func TestCleanUp(t *testing.T) {
 	assert.NoError(t, errCreatingNode)
 
 	// Set the NODE_NAME environment variable
-	os.Setenv("NODE_NAME", "node-1")
-	defer os.Unsetenv("NODE_NAME")
+	setEnvErr := os.Setenv("NODE_NAME", "node-1")
+	assert.NoError(t, setEnvErr, "error setting setenv")
+	defer func() {
+		err := os.Unsetenv("NODE_NAME")
+		assert.NoError(t, err)
```

Comment:
Maybe a useful message? `assert.NoError(t, err, "error unsetting NODE_NAME env variable")

Reply:
This is some unit test code so not sure if we really need this.